### PR TITLE
Export user management methods through JustInWrapper (including bug fix)

### PR DESCRIPTION
**Pull request by peiyaoh**
### Description

-

### Related Jira Tickets

-

### Comments 

-

Please ensure all items are complete before requesting a review:

- [ ] **Code Completed**: All code for the feature or bugfix has been implemented.
- [ ] **Unit Tests Added**: Unit tests are written for all new functionality, covering edge cases where applicable.
- [ ] **Integration Tests (where necessary)**: Integration tests are added for interactions between modules or components.
- [ ] **All Tests Passing**: All unit and integration tests are passing.
- [ ] **Type Safety Checked**: Ensure TypeScript types are correctly defined and used. Avoid `any` types without proper justification.
- [ ] **Typedoc Comments Added**: Ensure all functions, classes, interfaces, and types are documented using TypeDoc syntax.
- [ ] **Code Formatting**: All code files are formatted according to the project’s standards (e.g., Prettier, ESLint) and no linting errors remain.
- [ ] **No Console Logs**: Remove any `console.log` or other debugging output from the code.
- [ ] **Documentation Updated**: Update relevant project documentation (e.g., README, configuration guides) if necessary.

---

### CODE REVIEW

File: src/JustInWrapper.ts [hunk 1]
```diff
@@ -144,8 +144,58 @@ export class JustInWrapper {
...
   }
 
-  public async addUsersToDatabase(users: NewUserRecord[]) : Promise<void> {
-    await UserManager.addUsers(users);
+  /**
+   * Adds a list of new users to the database.
+   * @param {NewUserRecord[]} users - The list of new users to add.
+   * @returns {Promise<(JUser | null)[]>} The list of added users, or null if not found.
+   */
+  public async addUsersToDatabase(users: NewUserRecord[]) : Promise<(JUser | null)[]> {
```
Comment (numerator) [inline src/JustInWrapper.ts:152]: Why "ToDatabase"? It seems wordy without really adding anything.

File: src/JustInWrapper.ts [hunk 2]
```diff
@@ -144,8 +144,58 @@ export class JustInWrapper {
...
+    return await UserManager.addUsers(users);
+  }
+
+  /**
+   * Retrieves a list of users from the database.
+   * @returns {Promise<(JUser | null)[]>} The list of users, or null if not found.
+   */
+  public async getUsersFromDatabase(): Promise<(JUser | null)[]> {
```
Comment (numerator) [inline src/JustInWrapper.ts:160]: Why not "getAllUsers()"? It seems clearer and less wordy.

**Review by numerator — COMMENTED** [review]
Couple of naming questions. Otherwise looks good.

File: src/JustInWrapper.ts (hunk 2 above)
Comment (peiyaoh) [reply src/JustInWrapper.ts]: done.
